be/src/olap/tablet_manager.h

```diff
@@ -190,8 +192,12 @@ class TabletManager {
     typedef std::unordered_map<int64_t, TableInstances> tablet_map_t;
 
     // Protect _tablet_map, _partition_tablet_map, _shutdown_tablets
-    RWMutex _tablet_map_lock;
-    tablet_map_t _tablet_map;
+    int32_t _tablet_map_lock_shard_size;
+    RWMutex *_tablet_map_lock_array;
+    tablet_map_t *_tablet_map_array;
+
+    RWMutex _partition_tablet_map_lock;
+    RWMutex _shutdown_tablets_lock;
```

Comment:
And comment of these 2 locks: `_partition_tablet_map_lock` and `_shutdown_tablets_lock`.
Especially notice the lock order of these locks, to avoid dead lock.

Reply:
ok